src/Http/Response.php

```diff
@@ -40,6 +40,13 @@ class Response extends IlluminateResponse
     protected static $formatters = [];
 
     /**
+     * Array of formats' options.
+     *
+     * @var array
+     */
+    protected static $formats_options = [];
```

Comment:
Yep, that's a relevant remark. Replaced underscore_case in variables' names to camelCase. Thanks for reviewing!